apps/rush/typings/npm-package-arg/npm-package-arg.d.ts

```diff
@@ -0,0 +1,93 @@
+// Type definitions for npm-package-arg 5.1.2
+// Project: https://github.com/npm/npm-package-arg
+// Definitions by: pgonzal
+
+declare module 'npm-package-arg' {
+  namespace npmPackageArg {
+    interface IResult {
+      /**
+       * Indicates the type of dependency reference.  For example 'version' indicates
+       * a normal SemVer pattern.  See the package README.md for full docs.
+       *
+       * git - a git repository
+       * tag - a tagged version, e.g. "example@latest"
+       * version - A specific version number, e.g. "example@1.2.3"
+       * range - A version range, e.g. "example@2.x"
+       * file - A local .tar.gz, .tar or .tgz file
+       * directory - A local directory
+       * remote - An HTTP url to a .tar.gz, .tar or .tgz file
+       */
+      type: 'git' | 'tag' | 'version' | 'range' | 'file' | 'directory' | 'remote';
+      /**
+       * True for tag, version and range types.
+       */
+      registry: boolean;
+      /**
+       * If known, the "name" field expected in the resulting package.
+       */
+      name: string | null;
+      /**
+       * For scoped NPM packages, the scope name.
+       */
+      scope: string | null;
+
+      escapedName: string | null;
+
+      /**
+       * The original specifier passed to npmPackageArg.resolve(), or the specifier part of the
+       * argument for npmPackageArg().
+       */
+      rawSpec: string;
+
+      /**
+       * The normalized specifier.
+       */
+      saveSpec: string | null;
+
+      fetchSpec: string | null;
+
+      /**
+       * If set, this is a semver specifier to match against git tags with
+       */
+      gitRange: string | null;
+
+      /**
+       * If set, this is the specific committish to use with a git dependency.
+       */
+      gitCommittish: string | null;
+
+      hosted: string | null;
+
+      raw: string | null;
```

Comment:
>raw [](start = 6, length = 3)

The original un-modified string that was provided. If called as npa.resolve(name, spec) then this will be name + '@' + spec.